src/imars3dv2/filters/recon.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import numpy as np
+from tomopy.recon.algorithm import recon as tomo_recon
+
+
+def recon(
+    sinogram: np.ndarray,
```

Comment:
I would recommend changing the var name to `arrays` or `tomo` (folloing tomopy convention).

The same 3D arrays (stack of radiographs) can be a stack of sinograms if we look at it along the row axis.
Using the name sinogram here would imply the first axis is row whereas it should the rotation angle omega.